### PR TITLE
Feature/manifestation reserve button

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFSOEG-202
image

#### Description
We need to make sure to show the correct reservation button on each individual edition on the work page. This functionality has already been developed for the reservation button on top of the work page. This PR extends the existing components to be able to distinguish between 

#### Screenshot of the result 
image

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions
n/a

### CODE REVIEW

File: src/components/material/material-buttons/physical/MaterialButtonPhysical.tsx [hunk 1]
```diff
@@ -19,6 +21,21 @@ const MaterialButtonPhysical: FC<MaterialButtonPhysicalProps> = ({
...
+  if (isOnEditionCard) {
+    return (
+      <Button
+        label={t("reserveText")}
+        buttonType="none"
+        variant="filled"
+        disabled={false}
+        collapsible={false}
+        size="small"
+        onClick={() => {
```
[the remark this answers is not in the document]
Comment (Adamik10) [reply src/components/material/material-buttons/physical/MaterialButtonPhysical.tsx:33]: Good point :)

**Review by kasperg — CHANGES_REQUESTED** [review]
:+1: LGTM. There are a few things I would like you to look into.

File: src/components/material/material-buttons/MaterialButtons.tsx [hunk 2]
```diff
@@ -4,23 +4,36 @@ import {
...
 } from "../../../core/dbc-gateway/generated/graphql";
+import { convertPostIdToFaustId } from "../../../core/utils/helpers/general";
+import { Pid } from "../../../core/utils/types/ids";
 import MaterialButtonsOnline from "./online/MaterialButtonsOnline";
+import MaterialButtonsFindOnShelf from "./physical/MaterialButtonsFindOnShelf";
 import MaterialButtonsPhysical from "./physical/MaterialButtonsPhysical";
 
 export interface MaterialButtonsProps {
   manifestation: ManifestationsSimpleFieldsFragment;
+  size?: "large" | "medium" | "small" | "xsmall";
```
[the remark this answers is not in the document]
Comment (Adamik10) [reply src/components/material/material-buttons/MaterialButtons.tsx]: Great comment! :) done

File: src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import { FC } from "react";
+import { useGetAvailabilityV3 } from "../../../../core/fbs/fbs";
+import { useText } from "../../../../core/utils/text";
+import { Button } from "../../../Buttons/Button";
+
+export interface MaterialButtonsFindOnShelfProps {
+  size?: "large" | "medium" | "small" | "xsmall";
+  faustIds: string[];
```
[the remark this answers is not in the document]
Comment (Adamik10) [reply src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx]: Done! :) makes sense, thanks for suggesting it.

File: src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    return (
+      <span className="text-small-caption material-manifestation-item__find capitalize-all">
+        {t("materialIsLoanedOutText")}
+      </span>
+    );
+  }
+
+  return (
+    <span
+      className="link-tag text-small-caption material-manifestation-item__find capitalize-all"
```
[the remark this answers is not in the document]
Comment (Adamik10) [reply src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx:74]: yes ;)

**Review by sinejespersen — APPROVED** [review]

**Pull request merged**